src/Elm/Parser/Comments.elm

```diff
@@ -30,12 +31,11 @@ multilineCommentString : Parser.Parser String
 multilineCommentString =
     Parser.oneOf
         [ Parser.symbol "{-|"
-            |> Parser.backtrackable
-            |> Parser.map (\() -> problemUnexpected)
+            |> Parser.Extra.continueWith problemUnexpected
         , Parser.multiComment "{-" "-}" Nestable
-            |> Parser.mapChompedString (\comment () -> Parser.succeed comment)
+            |> Parser.getChompedString
         ]
-        |> Parser.andThen identity
+        |> Parser.backtrackable
```

Comment:
Does this make the whole `{- -}` backtrackable? I'm guessing that's only a performance problem if there is no closing `-}`?

Reply:
removed in favor of lookahead now